src/hektor/electrons/parabolic/semimaxwellian/ne00p_phiinfty_nobarriers.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+from scipy.special import erf
+from scipy.optimize import newton
+
+
+def ne00p_phiinfty_nobarriers(chi):
+    # MATLAB fzero(@(phiinfty) eq1(phiinfty)-eq2(phiinfty),-5);
+    # Ideal Python: root_scalar(eq1_minus_eq2, args=(chi,), x0=-5) does not pick Newton
+    # sol = root_scalar(eq1_minus_eq2, args=(chi,), x0=-5)
```

Comment:
I opened an issue about this in https://github.com/scipy/scipy/issues/12620

Reply:
There won't be changes in the near term regarding the behaviour of `newton` and `secant`. Options are:

a) Switch to a bracketing method with a suitable interval
b) Choose more appropriate initial guesses `x0` and `x1`
c) Write the derivative and choose better `x0`
d) Contribute a MATLAB-like solving method to SciPy (most probably, Octave based) and use it here